src/locales/en/translation.json

```diff
@@ -181,6 +181,23 @@
         "info": "See what you’ve visited with your GOV.UK One Login and when."
       }
     },
+    "addMfaMethod": {
+      "title": "Add another way to get security codes",
+      "sms": {
+        "title": "Backup phone number",
+        "hint": "Get a text message to a different mobile phone"
+      },
+      "app": {
+        "title": "Authenticator app for smartphone, tablet or computer",
+        "help": {
+          "title": "What is an authenticator app?",
+          "text1": "An authenticator app creates a security code that helps confirm it's you when you sign in.",
+          "text2": "You can use an authenticator app on your smartphone, tablet or desktop computer. Download an authenticator app for your smartphone or tablet from your app store or search online for an authenticator app for your computer."
+        }
+      },
+      "backLink": "I do not want to add another way to get security codes",
```

Comment:
Is the design correct that this doesn't end with a full stop?

Reply:
good point, i will clarify that